docs/components/motor/roboclaw.md

```diff
@@ -22,7 +22,10 @@ Follow [this guide](https://resources.basicmicro.com/roboclaw-motor-controllers-
 
 {{< /alert >}}
 
-To configure a `roboclaw` motor as a component of your machine:
+To use a `roboclaw` motor as a component of your machine, first wire your DC motor to the RoboClaw motor driver, and wire the driver to a board.
```

Comment:
@JessamyT this is what olivia said! the roboclaw is the motor driver. after editing this doc i realized we also could have known that from the doc itself oops...